content/docs/capabilities/ppl.mdx

```diff
@@ -304,6 +306,25 @@ allow:
         starts_with: 'admin@'
 ```
 
+### String List Matcher
+
+A string list matcher is an object that supports a single `has` operator as a key. The `has` operator checks that a given string is present in a list of strings.
+
+The `groups` and `record` criteria both support the `has` operator.
```

Comment:
the example below does not demonstrate `has` operator for `record`

Reply:
@wasaga I meant to add that. Thanks for catching it. I've updated the example.